Vienna/Sources/Main window/ArticleListView.m

```diff
@@ -126,9 +126,14 @@ -(void)initialiseArticleView
 	style.lineBreakMode = NSLineBreakByTruncatingTail;
 	style.tighteningFactorForTruncation = 0.0;
 	
-	reportCellDict = [[NSMutableDictionary alloc] initWithObjectsAndKeys:style, NSParagraphStyleAttributeName, nil];
-	unreadReportCellDict = [[NSMutableDictionary alloc] initWithObjectsAndKeys:style, NSParagraphStyleAttributeName, nil];
-		
+    if (@available(macOS 10.10, *)) {
+        reportCellDict = [[NSMutableDictionary alloc] initWithObjectsAndKeys:style, NSParagraphStyleAttributeName, [NSColor labelColor], NSForegroundColorAttributeName, nil];
+        unreadReportCellDict = [[NSMutableDictionary alloc] initWithObjectsAndKeys:style, NSParagraphStyleAttributeName, [NSColor labelColor], NSForegroundColorAttributeName, nil];
```

Comment:
I am wondering if it should be `labelColor` or `textColor`…
Probably the latter.